catalog/aws/ml-ops/s3.tf

```
@@ -65,50 +65,23 @@ resource "aws_s3_bucket" "model_store" {
   }
 }
 
-resource "aws_s3_bucket" "metadata_store" {
-  bucket = "${lower(var.name_prefix)}metadata-store-${local.random_bucket_suffix}"
-  acl    = "private"
-  tags   = var.resource_tags
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "AES256"
-      }
-    }
-  }
-}
-
-resource "aws_s3_bucket" "output_store" {
-  bucket = "${lower(var.name_prefix)}output-store-${local.random_bucket_suffix}"
-  acl    = "private"
-  tags   = var.resource_tags
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "AES256"
-      }
-    }
-  }
-}
-
 resource "aws_s3_bucket_object" "train_data" {
-  bucket = var.feature_store_override != null ? data.aws_s3_bucket.feature_store_override[0].id : aws_s3_bucket.feature_store[0].id
-  key    = "data/train/train.csv"
-  source = var.train_local_path
-
-  etag = filemd5(
-    var.train_local_path,
+  bucket       = var.feature_store_override != null ? data.aws_s3_bucket.feature_store_override[0].id : aws_s3_bucket.feature_store[0].id
+  key          = var.train_key
+  source       = var.train_local_path
+  content_type = var.content_type
+  for_each = fileset(
+    var.train_local_path, "var.train_key/*.png"
```

I think you meant to get:

```suggestion
    var.train_local_path, "${var.train_key}/*.png"
```

---

This has been updated.